### ISSUE TITLE
Electron 11 & sqlite3  support

### ISSUE THREAD

**Issue by twitharshil** [TryGhost/node-sqlite3#1409]
Hi there,
thanks for your great work on node-sqlite3. I use it for my electron-app!

I was just wondering if there is support for Electron 11 planned for the future. I a lot of warning messages when I try to rebuild the sqlite3 using the electron-rebuild latest version that is `2.3.4` as of today.

**Comment by hardik-simform** [comment]
I am getting the issue like FATAL ERROR: Error::Error napi_create_reference(https://github.com/mapbox/node-sqlite3/issues/1406)  with electron version 11 and  sqlite3 version 5.0.0.
 with electron version 8.3.4 and sqlite3 4.2.0 it was working.Please provide the solutions.

**Comment by jmartin-github** [comment]
Thank you for your dedication and great work on node-sqlite3.
I am using sqlite3 in my electron app, and I need to update it to electron 11, but it is not possible since sqlite3 is not supported in the latest versions of electron, only up to version 8.2 of electron, I was wondering if it is planned that in the next versions. Thanks in advance.

**Comment by justinjdickow** [comment]
+1

**Comment by zlshames** [comment]
I didn't want to do this, but I had to. I ended up switching to `better-sqlite` due to this compatibility error. Luckily, I was able to easily because I was utilizing TypeORM, so it was just a matter of changing a few strings to specify the new DB driver. I've had no other issues so far. So, if you are looking for an alternative, `better-sqlite` should work.

Also, as far as I know, `node-sqlite3` is only _technically_ compatible with Electron 8.x. I believe `better-sqlite` is compatible with the latest, v11
